descartes_core/src/cart_trajectory_pt.cpp

```diff
@@ -203,7 +211,78 @@ bool CartTrajectoryPt::getClosestJointPose(const std::vector<double> &seed_state
                                            const RobotModel &model,
                                            std::vector<double> &joint_pose) const
 {
-  NOT_IMPLEMENTED_ERR(false);
+  Eigen::Affine3d nominal_pose,candidate_pose;
+
+  if(!model.getFK(seed_state,candidate_pose))
+  {
+    return false;
```

Comment:
This kind of error is very unlikely but if it were to happen it'd be much easier to troubleshoot if we had an error message for this.